core/src/main/java/tachyon/Constants.java

```diff
@@ -57,6 +57,8 @@
   public static final int DEFAULT_BLOCK_SIZE_BYTE = 512 * MB;
 
   public static final int WORKER_BLOCKS_QUEUE_SIZE = 10000;
+  
+  public static final int DEFAULT_HOST_RESOLUTION_TIMEOUT = 5000;
```

Comment:
`DEFAULT_HOST_RESOLUTION_TIMEOUT_MS` should be more intuitive to know it's 5000ms instead of 5000s...